api/sales_dashboard/views.py

```diff
@@ -34,8 +32,6 @@
 
 OBJECTS_PER_PAGE = 50
 
-MAX_MIGRATABLE_IDENTITIES_SYNC = 1000
```

Comment:
I still think we should have an upper limit in the code. For self serve migrations at least. @dabeeeenster I know you said we could have a segment but I think we should 'belt and braces' it tbh.